Marlin/src/inc/Conditionals_post.h

```diff
@@ -149,19 +149,16 @@
     #define CORE_AXIS_2 C_AXIS
   #endif
   #define CORESIGN(n) (ANY(COREYX, COREZX, COREZY) ? (-(n)) : (n))
-#endif
-
-// Markforged kinematics
-#if ENABLED(MARKFORGED)
-  #define IS_MARKFORGED 1
-  #define MARKFORGED_AXIS_X A_AXIS
-  #define MARKFORGED_AXIS_Y B_AXIS
-  #define NORMAL_AXIS       Z_AXIS
+#elif ENABLED(MARKFORGED_XY)
+  // Markforged kinematics
+  #define CORE_AXIS_1 A_AXIS
```

Comment:
Regarding this naming: don't you think, it would be better to stick with _X and _Y prefixes? I am concerned because in CoreXY axises are fully interchangeable, but in case of MarkForged each axis has its own formula.

Reply:
Theoretically a "MarkForged" can be turned on its side to become an XZ or YZ arrangement, so the naming of these elements might as well be left open-ended. We'll continue to use "CORE" as our shorthand for axes that are combined into any differential arrangement.